backend/app/services/work_queue_service.py

```
@@ -0,0 +1,874 @@
+            logger.info(f"Deleted work queue: id={queue_id}, user_id={user_id}")
+
+            return True
+
+    def set_default_queue(self, user_id: int, queue_id: int) -> WorkQueueResponse:
+        """Set a queue as the default queue."""
+        with self.get_db() as db:
+            # First, unset all other default queues
+            db.query(Kind).filter(
+                Kind.user_id == user_id,
+                Kind.kind == self.WORK_QUEUE_KIND,
+                Kind.is_active == True,
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)